deploy/templates/nstemplatetiers/baseextendedidling/based_on_tier.yaml

```diff
@@ -0,0 +1,8 @@
+from:
```

Comment:
so, having such a single `based_on_tier.yaml` file will trigger the generation of `cluster.yaml`, `ns_dev.yaml`, `ns_stage.yaml` and `tier.yaml` for this `baseextendingidling` tier?

Reply:
not really - it doesn't generate any yaml files. 
This config will ensure creation/update of `NSTemplateTier` with the name `baseextendingidling` and related `TierTemplates` CRs

Reply:
ok, I see. So there is a risk that the NSTemplateTier generation fails during deployment (eg: missing template, etc.)? I assume this should be caught by our e2e tests?

Reply:
I believe that it should be already caught by unit-tests

Reply:
ok, we're good then